src/main/java/com/example/onlinebookstore/dto/shoppingcart/CartItemResponseDto.java

```diff
@@ -0,0 +1,9 @@
+package com.example.onlinebookstore.dto.shoppingcart;
+
+public record CartItemResponseDto(
+        Long id,
+        Long bookId,
+        String bookTitle,
+        int quantity
+) {
+}
```

Comment:
And this one should be just a simple dto
```suggestion
package com.example.onlinebookstore.dto.shoppingcart;

public record CartItemDto(
        Long id,
        Long bookId,
        String bookTitle,
        int quantity
) {
}
```